POLMED/src/components/footers/home-page/index.tsx

```diff
@@ -1,9 +1,21 @@
 import logoWhite from "@/img/logo/logo_white.png";
 import { Link } from "react-router-dom";
+import { useRef, useEffect } from "react";
 
 const HomePageFooter = () => {
+  const hpFooter = useRef<HTMLElement>(null);
+
+  useEffect(() => {
+    const height = hpFooter.current?.clientHeight;
+    const rootElement = document.querySelector(":root");
+
+    if (!height || !rootElement) return;
+
+    (rootElement as HTMLElement).style.setProperty("--hp-footer-height", `${height}px`);
+  }, []);
```

Comment:
a sprawdzałeś czy jeśli w trakcie korzystania ze stronki zmienisz szerokość ekranu, np w telefonie poprzez autoobrót, to stopka też się dostosuje?

Reply:
No niestety nie działa, w następnym commicie postaram się wszystko ogarnąć